index.html

```diff
@@ -22,17 +22,329 @@
+        <nav class="container25">
+            <ul class="data9">
+                <li class="Lang8"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
+    
+        <nav class="container26">
+            <ul class="navlink4">
+                <li class="project3"><a href="#">See this Project</a></li>
+                <li class="arrow2"><a href="#"><img src="imgformobile/Union.png" alt="sign"></a></li>
+            </ul>
+        </nav>
+    </nav>
+    
+    <nav class="container27">
+        <ul class="comp4">
+            <li class="rect4"><img src="imgformobile/Rectangle4.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container28">
+        <ul>
+            <li class="Proj4"><a href="#">Hello-World</a></li>
+        </ul>
+        <nav class="container29">
+            <ul class="data10">
+                <li class="Lang9"><a href="#">HTML/CSS</a></li>
+            </ul>
+        </nav>
+        <nav class="container30">
+            <ul class="data11">
+                <li class="Lang10"><a href="#">Ruby on </a></li>
+            </ul>
+        </nav>
+        <nav class="container31">
+            <ul class="data12">
+                <li class="Lang11"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
+    
+        <nav class="container32">
+            <ul class="navlink5">
+                <li class="project4"><a href="#">See this Project</a></li>
+                <li class="arrow3"><a href="#"><img src="imgformobile/Union.png" alt="sign"></a></li>
+            </ul>
+        </nav>
+    </nav>
+    
+    <nav class="container33">
+        <ul class="comp5">
+            <li class="rect5"><img src="imgformobile/Rectangle5.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container34">
+        <ul>
+            <li class="Proj5"><a href="#">Hello-World</a></li>
+        </ul>
+        <nav class="container35">
+            <ul class="data13">
+                <li class="Lang12"><a href="#">HTML/CSS</a></li>
+            </ul>
+        </nav>
+        <nav class="container36">
+            <ul class="data14">
+                <li class="Lang13"><a href="#">Ruby on </a></li>
+            </ul>
+        </nav>
+        <nav class="container37">
+            <ul class="data15">
+                <li class="Lang14"><a href="#">JavaScript</a></li>
+            </ul>
+        </nav>
+    
+        <nav class="container38">
+            <ul class="navlink6">
+                <li class="project5"><a href="#">See this Project</a></li>
+                <li class="arrow4"><a href="#"><img src="imgformobile/Union.png" alt="sign"></a></li>
+            </ul>
+        </nav>
+    </nav>
+    
+    <nav class="container39">
+        <ul class="comp6">
+            <li class="rect6"><img src="imgformobile/Rectangle6.png" alt="computer"></li>
+        </ul>
+    </nav>
+    <nav class="container40">
+        <ul>
+            <li class="Proj6"><a href="#">Hello-World</a></li>
```

Comment:
- [ ] Kindly change this to a paragraph tag because this will give ore information about the card  👍🏿